K8s/soda_scale_service.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+no_of_pods="$1"
+service_name="$2"
+
+if [ -z "$service_name" ] || [ -z "$no_of_pods" ]
+then
+        echo "Usage : ./soda_scale_service <service_name> <number_of_instances>"
+        exit 1
+fi
+echo "START :: Scaling Service " $2 to $1 "instances.."
```

Comment:
Can we use $no_of_pods and $service_name instead of $1 and $2?

Reply:
its internal to the script.. doesn't have any impact

Reply:
Done